CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Enhancements
 
+- Anonymize diff when updating a Kubernetes Secret object from salt state
+  (PR[#4313](https://github.com/scality/metalk8s/pull/4313))
```

Comment:
If you may just move it at the end of Enhancements (I think it's better to have the bumps first)